### PR TITLE
WIP: Register fastboot and ember-cli-fastboot versions

### PR DESCRIPTION
**Pull request by danmcclain**
Allows fastboot server to print out current versions on app boot

### CODE REVIEW

**Comment by danmcclain** [comment]
Output:
```
DEBUG: -------------------------------
DEBUG: Ember              : 2.7.3
DEBUG: Ember Data         : 2.8.1
DEBUG: FastBoot           : 1.0.0-rc.0
DEBUG: Ember CLI FastBoot : 1.0.0-beta.13
DEBUG: -------------------------------
Ember FastBoot running at http://[::]:3000
```

**Comment by danmcclain** [comment]
Try `FastBoot : 1.0.0-rc.0 (fastboot) / 1.0.0-beta.13 (ember-cli-fastboot)` as a format

**Comment by danmcclain** [comment]
We should only emit this in dev on browser side, and only emit cli-fastboot version in browser outside of dev

File: index.js [hunk 1]
```diff
@@ -140,6 +163,33 @@ module.exports = {
...
+  _getFastBootVersion: function() {
+    var VersionChecker = require('ember-cli-version-checker');
+    var checker = new VersionChecker(this);
+    var fastbootVersionChecker = checker.for('fastboot', 'npm');
+
+    return fastbootVersionChecker.version;
+  },
+
+  _getEmberCliFastBootVersion: function() {
+    var projectPath = process.cwd() + '/node_modules/ember-cli-fastboot';
```
Comment (stefanpenner) [inline index.js]: this isn't the correct path, as `ember s` can be anywhere within the root of your app.

File: addon/versions.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+
+if (Ember.libraries) {
+  Ember.libraries.register('FastBoot', 'FASTBOOT_VERSION');
```
Comment (stefanpenner) [inline addon/versions.js]: if we do this, we should mimic how ED does this:

* ember-data/version module is added
* that module is imported to populate https://github.com/emberjs/data/blob/c9d8212c857ca78218ad98d11621819b38dba98f/addon/-private/core.js#L26

**Pull request closed without merging**